drizzle/schema.ts

```diff
@@ -25,10 +26,79 @@ export const roster = sqliteTable(
   "roster",
   {
     id: integer("id").primaryKey({ autoIncrement: true }),
-    lastUpdate: integer("last_update", { mode: "timestamp" }),
+    lastUpdate: text("timestamp")
+      .notNull()
+      .default(sql`(current_timestamp)`),
     userId: text("user_id", { length: 255 }),
   },
   (t) => ({
     unq: unique().on(t.userId),
   }),
 );
+
+export const rosterArtifacts = sqliteTable(
+  "roster_artifacts",
+  {
+    id: integer("id").primaryKey({ autoIncrement: true }),
+    rosterId: integer("roster_id").references(() => roster.id, {
+      onDelete: "cascade",
+    }),
+    artifactId: integer("artifact_id"),
```

Comment:
lets be sure to add `notNull()` to the appropriate columns so that when we do use the drizzle built in types we don't have to deal with `| null` everywhere :)